portfolio/src/main/webapp/index.html

```diff
@@ -1,18 +1,73 @@
 <!DOCTYPE html>
 <html>
+    
   <head>
     <meta charset="UTF-8">
     <title>My Portfolio</title>
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
+    <!-- <style>
+         body {
+            background-image: url("/images.JPG");
+         }
+      </style> -->
   </head>
   <body>
-    <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
-    </div>
+      <!-- <img class = bgImage src = "/images/bg2.JPG"> -->
+       
+      <button id="play_button" onclick="loadIntro() ">Cynthia Ma</button>
+      <!-- <div id="greeting-container"></div> -->
+
+    <h1 id="introTitle" style="display:none">Cynthia's Portfolio</h1>
+      <p  id="introInfo" style="display:none">Hi I'm Cynthia. I'm a sophomore studying Computer Science and Statistics at Cornell University. 
+          I like photography and playing tennis and traveling. I was born and raised in New York and spend most of my time 
+          taking pictures, hanging out with my friends, or learning new things. I like to read and do my best to keep reading even 
+          when life gets hectic. 
+      </p>
+
+      <button id="profesh1" class="profesh" onclick=" window.open('https://github.com/cynthia-m','_blank')" style="display:none">Github</button>
+      <button id="profesh2" class = "profesh" onclick=" window.open('https://www.linkedin.com/in/cma/','_blank')" style="display:none">LinkedIn</button>
+      <button id="profesh3" class = "profesh" onclick="getPhotos_prev()" style ="display:none">Photography</button>
+    <!-- <button id="profesh5" class = "profesh"  style ="display:none">Test</button> -->
+
+        <!-- use classes to streamline -->
+        <!-- <button id="back" onclick="loadIntro()" style="display:none">Back</button> -->
+    <p id = "photo_intro" style="display:none"> As I mentioned, I like photography! I like to travel and like photography for capturing memories of
+          all the beautiul places I've visited. </p>
+          <!-- should not be categorized as pics -->
+          <button id="prev" class = "next" onclick="getPhotos_prev()" style ="display:none">Prev</button>
+      <img class = "pics" style="display:none" src = "/images/yellow-roses.jpg" alt="Yellow Roses in Portland, OR, USA" >
+      <img class = "pics" style="display:none" src = "/images/IMG_1204.jpg" alt="Sunrise in Canada" >
+      <img class = "pics" style="display:none" src = "/images/IMG_0610.JPG" alt="Portland, OR, USA Lan Su Garden">
+      <img class = "pics" style="display:none" src = "/images/Copy of IMG_0468.JPG" alt="Musee d'Orsay Clock" >
```

Comment:
This is definitely a small thing, but naming your images more consistently will be a long-term win. Instead of using names like IMG_1204.jpg, use "sunrise_in_canada.jpg" instead, it will be easier to debug and reason about! :)